component-classes/index.js

```diff
@@ -107,15 +107,15 @@ export const steps = {
 
 export const card = {
   card: 'cursor-pointer overflow-hidden relative transition-all',
-  cardShadow: 'rounded-8 s-surface-elevated-200 hover:s-surface-elevated-200-hover',
+  cardShadow: 'group rounded-8 s-surface-elevated-200 hover:s-surface-elevated-200-hover active:s-surface-elevated-200-active',
+  cardUnselected: '', // TODO: Remove in v2 - Not used after introducing surface
```

Comment:
@magnuh shouldn't we keep those component classes for backwards compatibility, like it was mentioned in another similar rewrite-PR? https://github.com/warp-ds/css/pull/180#discussion_r1531829325

Reply:
Which ones are you referring to?
The ones that are not present somewhere else should have been added to `surfaceUpdates` below, aren't they?

Reply:
Oh, then I might have got confused by the fact that some classes are moved to `backwardsCompatibleClasses`, and some are kept where they were with a "todo" comment. I thought that in the latter cases, it was because updating the css package could break something lower versions of component packages. But that probably doesn't make sense to cater for, because there's been more changes to other component classes, that would break styles anyway for those components.

Reply:
For the case of `card.cardUnselected`, we can't remove this without bumping the major version, as it would be a true breaking change if someone uses it and updates their dependency.
For the case of removing unocss classes in these strings, we are safe, as long as these removed classes are either present in some other strings OR moved to the list of `backwardsCompatibleClasses`.